src/V8JsiRuntime.cpp

```diff
@@ -515,10 +515,20 @@ V8Runtime::V8Runtime(V8RuntimeArgs &&args) : args_(std::move(args)) {
   v8::Context::Scope context_scope(context);
 
 #if defined(_WIN32) && defined(V8JSI_ENABLE_INSPECTOR)
+  void* inspector_agent = isolate_->GetData(ISOLATE_INSPECTOR_SLOT);
+  if (inspector_agent) {
+    inspector_agent_ = reinterpret_cast<inspector::Agent*>(inspector_agent)->getShared();
+  } else {
+    inspector_agent_ = std::make_shared<inspector::Agent>(

```

Comment:
>inspector_agent_ [](http://example.com/codeflow?start=4&length=16)

I presume this doesn't create network sockets until started, right? The memory impact of always creating an inspector agent for any runtime should be pretty minimal but it might be worth taking a quick look

Reply:
Thanks Tudor for taking a quick look.. 
Yes, we are now only initializing the inspector client, not starting the network sockets. I've done some preliminary analysis on the performance impacts.